README.md

```diff
@@ -70,76 +68,72 @@ Click 'set me up!' on https://bintray.com/gov-uk-notify/maven/notifications-java
 
 ## Getting started
 
-**Import the `NotificationClient`**
 
 ```java
 import uk.gov.service.notify.NotificationClient;
 import uk.gov.service.notify.Notification;
 import uk.gov.service.notify.NotificationList;
 import uk.gov.service.notify.NotificationResponse;
+
+NotificationClient client = new NotificationClient(api_key, serviceId, "https://api.notifications.service.gov.uk");
 ```
 
-**Create a new instance of `NotificationClient` and objects returned by the client**
+Generate an API key by signing in to
+[GOV.UK Notify](https://www.notifications.service.gov.uk) and going to
+the **API integration** page.
+
+You'll find your service ID on the **API integration** page.
+
+## Send a message
+
+Text message:
 
 ```java
-NotificationClient client = new NotificationClient(api_key, serviceId, "https://api.notifications.service.gov.uk");
+NotificationResponse response = client.sendSms(templateId, mobileNumber, personalisation);
```

Comment:
Checked with @servingUpAces, made the change in 39744b1b710b33d885289cc77d88abab0934a06b